lib/assembly-objectfile/content_metadata/file.rb

```diff
@@ -13,7 +13,7 @@ class File
         'image/tif' => { preserve: 'yes', shelve: 'no', publish: 'no' },
         'image/tiff' => { preserve: 'yes', shelve: 'no', publish: 'no' },
         'image/jp2' => { preserve: 'no', shelve: 'yes', publish: 'yes' },
-        'image/jpeg' => { preserve: 'yes', shelve: 'no', publish: 'no' },
+        'image/jpeg' => { preserve: 'yes', shelve: 'yes', publish: 'no' },
```

Comment:
If other non-3d objects have a jpeg file, this could now get shelved, but it wasn't before.  Do we need to guard against that? Are there things expecting this behavior?

Reply:
Yes, this is true - Andrew and I discussed this and decided that the actual solution is more than just a guard clause for 3d objects, but rather a more comprehensive way to specify these attributes per accessioning batch as opposed to by mimetype.  This was possible in the legacy pre-assembly app, and is something that could be ported above.  He thought it was low risk to make this change for now to support 3D.